faithcomesbyhearing/sofria-cli
# FCBHDBP-436 modify sofria-cli

# Description
It has updated the sofria_mediaId.js file to achive to create the .json files from usx directory. It will read each .usx file from a specific directory and it will create a .json file for each chapter.

E.g. I have used the folder Spanish_N2SPNTLA_USX and I have executed the following command:

```shell
node sofria_mediaId.js ~/fcbh-etl-local/Spanish_N2SPNTLA_USX
```
```shell
pattern:
node sofria_mediaId.js [PATH_USX_FOLDER]
```

I have added the parameters: offset and limit to process a limit of files. E.g. about the Spanish_N2SPNTLA_USX folder the two first files are:  001GEN.usx and 002EXO.usx

The output of the above command was:

```shell
## pwd: output/Spanish_N2SPNTLA_USX-json/

001GEN_001.json  001GEN_010.json  001GEN_019.json  001GEN_028.json  001GEN_038.json  001GEN_047.json  002EXO_006.json  002EXO_015.json  002EXO_024.json  002EXO_034.json  002EXO_044.json
001GEN_002.json  001GEN_011.json  001GEN_020.json  001GEN_030.json  001GEN_039.json  001GEN_048.json  002EXO_007.json  002EXO_016.json  002EXO_025.json  002EXO_035.json  002EXO_045.json
001GEN_003.json  001GEN_012.json  001GEN_021.json  001GEN_031.json  001GEN_040.json  001GEN_049.json  002EXO_008.json  002EXO_017.json  002EXO_026.json  002EXO_036.json  002EXO_046.json
001GEN_004.json  001GEN_013.json  001GEN_022.json  001GEN_032.json  001GEN_041.json  001GEN_050.json  002EXO_009.json  002EXO_018.json  002EXO_027.json  002EXO_037.json  002EXO_047.json
001GEN_005.json  001GEN_014.json  001GEN_023.json  001GEN_033.json  001GEN_042.json  002EXO_001.json  002EXO_010.json  002EXO_019.json  002EXO_028.json  002EXO_038.json  002EXO_048.json
001GEN_006.json  001GEN_015.json  001GEN_024.json  001GEN_034.json  001GEN_043.json  002EXO_002.json  002EXO_011.json  002EXO_020.json  002EXO_030.json  002EXO_039.json  002EXO_049.json
001GEN_007.json  001GEN_016.json  001GEN_025.json  001GEN_035.json  001GEN_044.json  002EXO_003.json  002EXO_012.json  002EXO_021.json  002EXO_031.json  002EXO_041.json  002EXO_050.json
001GEN_008.json  001GEN_017.json  001GEN_026.json  001GEN_036.json  001GEN_045.json  002EXO_004.json  002EXO_013.json  002EXO_022.json  002EXO_032.json  002EXO_042.json
001GEN_009.json  001GEN_018.json  001GEN_027.json  001GEN_037.json  001GEN_046.json  002EXO_005.json  002EXO_014.json  002EXO_023.json  002EXO_033.json  002EXO_043.json
```
The above files were created inside folder: `./output/Spanish_N2SPNTLA_USX-json/`

## Issue Link
https://fullstacklabs.atlassian.net/browse/FCBHDBP-436

## How Do I QA This
- Download the local USX files into folder:` ~/fcbh-etl-local/`
- Run: `node sofria_mediaId.js ~/fcbh-etl-local/Spanish_N2SPNTLA_USX 0 2`
- Check that json files are created inside folder: `./output/Spanish_N2SPNTLA_USX-json/`. There must be 66 files
